libmamba/include/mamba/core/output.hpp

```diff
@@ -54,6 +55,11 @@ namespace mamba
             right,
         };
 
+        const std::map<alignment, const char*> alignmentMarkers = {
```

Comment:
I think we can use `unordered_map` here instead of `map` (for speed) if that's not posing any problem.

Reply:
Good point, changed in 39b13444